src/main.js

```diff
@@ -1,4 +1,4 @@
-import { createApp } from 'vue'
+import {createApp} from 'vue'
 import App from './App.vue'
 
-createApp(App).mount('#app')
+createApp(App).mount(document.body)
```

Comment:
Rendering the app directly on the body allows for using the default HTML file of the `HtmlWebpackPlugin`. Otherwise, a template file would have had to be provided.

This way, I could get rid of the entire `public` folder for now, since we don't have any other static resources.